docs/docs/user-guide/training-data-format.mdx

```diff
@@ -0,0 +1,659 @@
+---
+id: training-data-format
+sidebar_label: Training Data Format
+title: Training Data Format
+description: Description of the YAML format for training data
+---
+
+## Overview
+
+### YAML Format
+
+Rasa Open Source 2.0 uses [YAML](https://yaml.org/spec/1.2/spec.html) as
+a unified and extendable way to manage all training data,
+including NLU data, stories and rules. 
+
+With the YAML format, training data can be split over any number of YAML files,
+and every file can contain any kind of data. 
```

Comment:
@degiz doing the multimedia response implementation I noticed that this is actually not quite true (we should make it work though). We do the "what is this file type containing" discovery (is it nlu, stories, ...) and the parsing afterwards. if your file contains stories and nlu data, only one of them will be read. are you aware of that?

Reply:
If a file contains `nlu` and `stories` key, it should be accepted by both the NLU and Stories parser.
If that's not the case, I need to fix that 😬 I'll check that.

Reply:
mhm yes I think both will accept them, but I though only one of them will actually get called 🤔  - I'll close this comment though nothing we need to discuss here.